spring-integration-core/src/main/java/org/springframework/integration/transaction/IntegrationResourceHolderSynchronization.java

```diff
@@ -24,11 +24,13 @@
  * @author Artem Bilan
  * @since 4.0
  */
-public abstract class IntegrationResourceHolderSynchronization
+public class IntegrationResourceHolderSynchronization
```

Comment:
Since you did some changes in this class, add your name to the `@author` and update Copyright to the actual year. In all affected classes